SDRNA RNA Mining.ipynb

```diff
@@ -0,0 +1,278 @@
+      "\n",
+      "type: rRNA\n",
+      "location: [482044:482163](+)\n",
+      "qualifiers:\n",
+      "    Key: db_xref, Value: ['SGD:S000029708']\n",
+      "    Key: experiment, Value: ['EXISTENCE:direct assay:GO:0003735 structural constituent of ribosome [PMID:16047201]', 'EXISTENCE:direct assay:GO:0022625 cytosolic large ribosomal subunit [PMID:16047201]', 'EXISTENCE:mutant phenotype:GO:0002181 cytoplasmic translation [PMID:16047201]']\n",
+      "    Key: locus_tag, Value: ['RDN5-4']\n",
+      "    Key: note, Value: ['Variant 5S ribosomal rRNA (5S rRNA) gene; located at a site distal to the RDN1 locus; unable to support cell viability when provided as sole source of 5S rRNA; identical in sequence to variant alleles RDN5-2, RDN5-3, and RDN5-5']\n",
+      "    Key: product, Value: ['5S ribosomal RNA']\n",
+      "\n",
+      "type: rRNA\n",
+      "location: [485696:485815](+)\n",
+      "qualifiers:\n",
+      "    Key: db_xref, Value: ['SGD:S000029710']\n",
+      "    Key: experiment, Value: ['EXISTENCE:direct assay:GO:0003735 structural constituent of ribosome [PMID:16047201]', 'EXISTENCE:direct assay:GO:0022625 cytosolic large ribosomal subunit [PMID:16047201]', 'EXISTENCE:mutant phenotype:GO:0002181 cytoplasmic translation [PMID:16047201]']\n",
+      "    Key: locus_tag, Value: ['RDN5-5']\n",
+      "    Key: note, Value: ['Variant 5S ribosomal rRNA (5S rRNA) gene; located at a site distal to the RDN1 locus; unable to support cell viability when provided as sole source of 5S rRNA; identical in sequence to variant alleles RDN5-2, RDN5-3, and RDN5-4']\n",
+      "    Key: product, Value: ['5S ribosomal RNA']\n",
+      "\n",
+      "type: rRNA\n",
+      "location: [489348:489469](+)\n",
+      "qualifiers:\n",
+      "    Key: db_xref, Value: ['SGD:S000029711']\n",
+      "    Key: experiment, Value: ['EXISTENCE:direct assay:GO:0003735 structural constituent of ribosome [PMID:16047201]', 'EXISTENCE:direct assay:GO:0022625 cytosolic large ribosomal subunit [PMID:16047201]', 'EXISTENCE:mutant phenotype:GO:0002181 cytoplasmic translation [PMID:16047201]']\n",
+      "    Key: locus_tag, Value: ['RDN5-6']\n",
+      "    Key: note, Value: ['Variant 5S ribosomal rRNA (5S rRNA) gene; located at a site distal to the RDN1 locus']\n",
+      "    Key: product, Value: ['5S ribosomal RNA']\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "Purpose: mine the rRNA from Genbank sequences\n",
+    "and output in a fasta file with the accession number, locus tag, \n",
+    "the start, the end, and the direction of the sequence, with\n",
+    "optional database identifiers and optional prodcuts or notes\n",
+    "\n",
+    "The current version looks for the .gb Genbank files in the folder\n",
+    "that is two levels under the main folder\n",
+    "and loops through each Genbank files to look for the rRNA feature.\n",
+    "\"\"\"\n",
+    "\n",
+    "def SDRNA_RNA_Mining(folder,subfolder):\n",
```

Comment:
I'd suggest to create a Py library to build core functions frequently used in the Jupyther NB:
- that will provide code modularity
- and it will also help code review activities

Reply:
Added [selectfiles.ipynb](https://github.com/Zymergen/adv-tech-prog/blob/FZL_merge_on_ATP/Code/selectfiles.ipynb) 
to [rRNA_mining.ipynb](https://github.com/Zymergen/adv-tech-prog/blob/FZL_merge_on_ATP/Code/rRNA_mining.ipynb) on atp-tech-prog --> FZL_merge_on_ATP branch --> Code.